HISTORY.rst

```diff
@@ -1,6 +1,42 @@
 History
 =======
 
+0.8.1 -- 2019-01-28
+-------------------
+
+More information about this release can be found on the `0.8.1 milestone`_.
+
+New Features
+~~~~~~~~~~~~
+
+- Add X509 Adapter that can handle PKCS12 
```

Comment:
Since there's new features the release should be `0.9.0`.

Also if you can make the date the 29th or the 30th, that'd be perfect. (I like the date in the history to match the date that I actually release it).

Reply:
Also we should bump the version of the actual package (in `requests_toolbelt/__init__.py` or there-abouts) to match the version here.